src/components/appLoadTime/appLoadTime.tsx

```diff
@@ -11,75 +20,142 @@ export const AppLoadTime = () => {
 
   return (
     <div data-qa="utils.appLoadTime.container">
-      <Card title="App Load Time Utilities" style={{ width: "auto" }}>
-        <Flex gap={5}>
-          <Button data-qa="utils.appLoadTime.get.btn" onClick={showWidget}>
-            utils.appLoadTime.Get/MarkEnd
-          </Button>
-          <Button
-            data-qa="utils.appLoadTime.getEntries.btn"
-            onClick={async () => {
-              const resultString = platformProps.utils.appLoadTime
-                .getEntries()
-                .map((entry) => {
-                  return `Name: ${entry.name}, Entry Type: ${entry.entryType}, Start Time: ${entry.startTime}, Duration: ${entry.duration}`;
-                })
-                .join("; ");
-              setEntries(resultString);
-            }}
-          >
-            utils.appLoadTime.getEntries
-          </Button>
-          <Button
-            data-qa="utils.appLoadTime.mark.btn"
-            onClick={() => {
-              platformProps.utils.appLoadTime.markStart(
-                "iLove1FESoMuchMarkTest",
-              );
-              setMeasure("Mark started");
-            }}
-          >
-            utils.appLoadTime.markStart
-          </Button>
-          <Button
-            data-qa="utils.appLoadTime.measure.btn"
-            onClick={() => {
-              const result = platformProps.utils.appLoadTime.markEnd(
-                "iLove1FESoMuchMarkTest",
-              );
+      <UtilitySection
+        title="Performance Monitoring Utilities"
+        description="Measure and track application load times, and custom timing markers"
+      >
+        <Card style={utilityCard}>
+          <Flex {...flexProps}>
```

Comment:
[nit] This is usually a good candidate for a custom component that wraps `<Flex>` like - 

```tsx
function FlexCustom = (props) => <Flex {...flexProps, ...props} />
```

That way you can use `<FlexCustom>` wherever thats needed.